packages/docs/src/locales/en/en.json

```diff
@@ -685,7 +685,9 @@
         "immediateValidation": "Sets the validation to be performed when the component is mounted",
```

Comment:
Note should stay on the same line as count.

<img width="206" alt="image" src="https://user-images.githubusercontent.com/5394573/176714239-5ff175d9-b2f7-4974-b931-82f30a24319a.png">